packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.scss

```diff
@@ -180,6 +180,7 @@
 
   & button {
     width: 100%;
-    padding-inline-end: 0;
+    padding-inline: 15px 0;
+    padding-block: calc(0.875rem - 3px);
```

Comment:
Could you explain this calculation, as in where these numbers come from?

Reply:
@ibacher It's taken from [here](https://github.com/openmrs/openmrs-esm-core/blob/43a20b2cc5673071ed9c816c20a93682cb9e977e/packages/framework/esm-styleguide/src/_overrides.scss#L385), and seems it also can be simplified to `spacing.$spacing-04`, the result doesn't change.

![2024-01-10 17 21 30](https://github.com/openmrs/openmrs-esm-patient-chart/assets/17073192/9ca2ac49-ce3c-487b-a235-5b6b094fddab)